examples/file/file.out

```diff
@@ -0,0 +1,4 @@
+# To run this sample, navigate to the directory that contains the
+# `.bal` file and issue the `ballerina run` command.
+$ ballerina run file.bal
+
```

Comment:
Shouldn't we display the output of the example here?

Reply:
Output value depends on where the user executes the example. we can add sample output.

Reply:
Fixed in 0516bf0